src/sinks/datadog/metrics/encoder.rs

```diff
@@ -419,6 +436,14 @@ fn sketch_to_proto_message(
     let k = bins.into_iter().map(Into::into).collect();
     let n = counts.into_iter().map(Into::into).collect();
 
+    let event_metadata = metric.metadata();
+    let metadata = generate_sketch_metadata(
+        event_metadata.datadog_origin_metadata(),
+        event_metadata.source_type(),
+    );
+
+    debug!("Generated sketch metadata: {:?}", metadata);
```

Comment:
Just a note to remove this or bump it down to trace level.

Reply:
Do we have any published guidelines on deciding log levels?
I see https://github.com/vectordotdev/vector/blob/master/docs/DEVELOPING.md#logging-style , but that doesn't cover this subject.

Fresh on my mind is the lack of useful debug logs from recent issues.

Reply:
Unfortunately I don't think we have documented guidance; currently it is a bit of an art. In this case we would end up logging for every encoded sketch which seems pretty high volume to me to be included at the debug level. I could see it at the trace level though.